### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by smell-of-curry**
Updated Lots of Looks/grammar errors, Readme Looks a lot better Listing Autors Download description, Features, Acknowledgements And more.

### CODE REVIEW

**Comment by Intoprelised** [comment]
i hack on ur realm :D

**Comment by hrkyr** [comment]
this is so shit and im laughing so hard

**Comment by Intoprelised** [comment]
ikr, puts half the contributors

**Comment by hrkyr** [comment]
this is the worst read me i have ever seen

**Comment by smell-of-curry** [comment]
> this is the worst read me i have ever seen

lol how its just a read me get tf out of here

**Comment by hrkyr** [comment]
> > this is the worst read me i have ever seen
> 
> 
> 
> lol how its just a read me get tf out of here

yeah and you managed to make it look ugly..

**Comment by smell-of-curry** [comment]
> > > this is the worst read me i have ever seen
> > 
> > 
> > lol how its just a read me get tf out of here
> 
> yeah and you managed to make it look ugly..

you cant see what it looks like until its accepted

**Comment by smell-of-curry** [comment]
> > > > this is the worst read me i have ever seen
> > > 
> > > 
> > > lol how its just a read me get tf out of here
> > 
> > 
> > yeah and you managed to make it look ugly..
> 
> you cant see what it looks like until its accepted

nvm
didn't know u could do that

**Comment by hrkyr** [comment]
you said you removed spelling errors but you added more lol

**Comment by hrkyr** [comment]
you also removed buttons which just renders build status useless

**Comment by smell-of-curry** [comment]
> you said you removed spelling errors but you added more lol

like what

**Comment by Intoprelised** [comment]
other then doing the changes uself how do i see them

> > > this is the worst read me i have ever seen
> > 
> > 
> > lol how its just a read me get tf out of here
> 
> yeah and you managed to make it look ugly..
How can I tell without copying the entire file to a readme.md?

File: README.md [hunk 1]
```diff
@@ -1,25 +1,93 @@
...
+  
+## Authors
+
+- [@GodGamer029](https://github.com/GodGamer029)
+- [@SurvirvorHRG](https://github.com/SurvirvorHRG)
+- [@richardletshacks](https://github.com/richardletshacks)
+- [@FloppyDolphin57](https://github.com/FloppyDolphin57)
+- [@grajkogj](https://github.com/grajkogj)
+- [@CortexPE](https://github.com/CortexPE)
+- [@Johnmacrocraft](https://github.com/Intoprelised)
```
Comment (Intoprelised) [inline README.md:67]: Yes this is accurate 100%

Comment (Intoprelised) [reply README.md:67]: totally

**Pull request closed without merging**